lib/updaters/types/yaml.js

```diff
@@ -0,0 +1,11 @@
+const YAML = require('yaml')
+
+module.exports.readVersion = function (contents) {
+  return YAML.parse(contents).version
+}
+
+module.exports.writeVersion = function (contents, version) {
+  const yaml = YAML.parse(contents)
```

Comment:
```js
const yaml = YAML.parseDocument(contents);
yaml.set('version', version);
return yaml.toString();
```

would be better as it would preserve the original comments, formatting etc.